src/main/webapp/assets/modules/get-promoted/gp-form.mod.js

```diff
@@ -0,0 +1,28 @@
+techlooper
+  .directive("getPromotedForm", function ($http, vnwConfigService) {
+    return {
+      restrict: "E",
+      replace: true,
+      templateUrl: "modules/get-promoted/gp-form.tem.html",
+      link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.